core/frontend/src/store/ardupilot_capabilities.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  getModule,
+Module, VuexModule,
+} from 'vuex-module-decorators'
+
+import store from '@/store'
+import autopilot_data from "./autopilot"
+
+@Module({
+  dynamic: true,
+  store,
+  name: 'ardupilot_capabilities',
+})
+
+class ArdupilotCapabilitiesStore extends VuexModule {
+
+  get firmware_supports_actuators(): boolean {
+    return autopilot_data.parameter('ACTUATOR1_INC') !== undefined
+  }
+
+  get firmware_supports_light_functions(): boolean {
+    // TODO: light functions were introduced at the same time as actuators
+    return autopilot_data.parameter('ACTUATOR1_INC') !== undefined
```

Comment:
Is there pre-filtering somewhere, that makes this apply only to ArduSub?
IIRC Actuator outputs are more general ArduPilot, but Lights is a Sub thing - I could be wrong on both accounts though 🤷‍♂️